setup.py

```diff
@@ -275,8 +275,8 @@ def read_requirements(path="requirements/run.txt"):
       packages=[],
       extras_require={
           'dev': [
-              'pytest==7.0.0',
-              'pytest-cov==3.0.0',
+              'pytest==8.0.1',
```

Comment:
@Alopalao, trivial detail, but `extra_require` `dev` can be nuked on NApps, so this one here was leftover from prior refactoring. Currently it's already installing with [-e git+https://github.com/kytos-ng/kytos.git#egg=kytos[dev]](https://github.com/kytos-ng/kytos_stats/blob/master/requirements/dev.in#L3), so it'll fetch the dev dependencies from kytos `extra_require` `dev`.

This can also be done in another issue/PR since it won't cause issues.

Reply:
@Alopalao if you can capture this in another issue ^, trivial stuff but let's also nuke this

Reply:
[Issue](https://github.com/kytos-ng/kytos_stats/issues/14) created